### ISSUE TITLE
Support for analyzer nuget

### ISSUE THREAD

**Issue by sailro** [praeclarum/FuGetGallery#133]
See the following nuget:
https://www.fuget.org/packages/Microsoft.Unity.Analyzers

This is a special nuget for a Roslyn Analyzer (so the interesting part is that an `analyzers` folder is used instead of the expected `lib` one)

So `fuget` is currently confused to display/diff any assembly/type info.

File hierarchy for this nuget:

```
.signature.p7s
Microsoft.Unity.Analyzers.nuspec
[Content_Types].xml
analyzers\dotnet\cs\Microsoft.Unity.Analyzers.dll
package\services\metadata\core-properties\f9366dce472b4a0fb34e4ed667c83570.psmdcp
tools\install.ps1
tools\uninstall.ps1
_rels\.rels
```

So it's probably necessary to review those code parts:
https://github.com/praeclarum/FuGetGallery/search?q=%22lib%22

Documentation:
https://docs.microsoft.com/en-us/nuget/guides/analyzers-conventions#analyzers-path-format

**Comment by praeclarum** [comment]
Right, right, thanks. Yeah I need better support for Tool nugets too.
